controllers/machinepools.go

```diff
@@ -43,48 +53,100 @@ func scaleMachinePool(aw *arbv1.AppWrapper, userRequestedInstanceType string, re
 	createMachinePool, _ := cmv1.NewMachinePool().ID(machinePoolID).InstanceType(userRequestedInstanceType).Replicas(replicas).Labels(m).Build()
 
 	klog.Infof("Create machinepool with instance type %v and name %v", userRequestedInstanceType, createMachinePool.ID())
-	clusterMachinePools.Add().Body(createMachinePool).SendContext(context.Background())
-}
+	response, err := clusterMachinePools.Add().Body(createMachinePool).SendContext(context.Background())
+	if err != nil {
+		return fmt.Errorf("failed to add machine pool: %v", err)
```

Comment:
change fmt to logger

Reply:
Hey @asm582,

Apologies, but just to clarify are you suggesting to only log the error message rather than return it, or switch the implementation to use a package named "logger"

Reply:
please use a logger package